README.rst

```diff
@@ -62,13 +62,17 @@ These templates are meant to be used inside a package which was created by the a
 - viewlet
 - vocabulary
 
+
 Compatibility
 =============
 
 Add-ons created with ``bobtemplates.plone`` are tested to work in Plone 4.3.x and Plone 5.
 They should also work with older versions but that was not tested.
 It should work on Linux, Mac and Windows.
 
+.. note::
+    Please note that the theme tempates by default only work for Plone > 5, because they are based on Barceloneta. But they can be use for older version too, with some changes to the rules file.
```

Comment:
```suggestion
    Please note that the theme templates by default only work for Plone > 5, because they are based on Barceloneta. But they can be use for older version too, with some changes to the rules file.
```